examples/field_lines.py

```diff
@@ -55,7 +57,15 @@
                  frame=gong_map.coordinate_frame)
 field_lines = tracer.trace(seeds, output_)
 
-# We plot the field lines
-plotter.plot_field_lines(field_lines)
+
+# We cal also specify a color function while plotting the field lines.
```

Comment:
```suggestion
# We can also specify a color function while plotting the field lines.
```

Reply:
Could you also add a bit more description of what this function needs to look like? Something like
> This function takes a single field line, and returns a color. In this case we use a Matplotlib norm and colormap to return a tuple of RGBA values.

Reply:
Sure thing!